src/config/client.ts

```diff
@@ -1,10 +1,9 @@
 import axios from 'axios';
 
-const baseUrl = process.env.NEXT_PUBLIC_API_URL || 'http://localhost:8001/api/v1';
```

Comment:
I think we should keep the localhost conditional rather than require the .env property for local testing, especially considering we currently only have one variable on the frontend.  Thoughts?